### PR TITLE
Options style support customize & update README.md

### PR DESCRIPTION
**Pull request by GeoffZhu**
fix issue #220

### CODE REVIEW

**Comment by yesmeck** [comment]
Could you add a testcase?

File: src/Plugin.js [hunk 1]
```diff
@@ -58,8 +58,8 @@ export default class Plugin {
       this.selectedMethods[methodName] = addDefault(file.path, path, { nameHint: methodName });
       if (style === true) {
         addSideEffect(file.path, `${path}/style`);
-      } else if (style === 'css') {
-        addSideEffect(file.path, `${path}/style/css`);
+      } else if (typeof style === 'string') {
```
Comment (afc163) [inline src/Plugin.js]: how about a function?

**Comment by GeoffZhu** [comment]
Testcase added

File: src/Plugin.js (hunk 1 above)
Comment (afc163) [reply src/Plugin.js]:
```js
style: (name) => `antd/lib/${name}/style`
```

like https://github.com/ant-design/babel-plugin-import/pull/163

Comment (GeoffZhu) [reply src/Plugin.js]: fixed

**Comment by GeoffZhu** [comment]
@yesmeck @afc163 
I try 1.7.0 in my project. There was a serious mistake. I'm so sorry about than.

Reason: 
In [Babel docs of .babelrc](http://babeljs.io/docs/usage/babelrc/)
> All Babel API options except the callbacks are allowed (because .babelrc files are serialized as JSON5).

So, make option `style` as a Function lead to some bugs. Babel will throw Error 'Not parse JSON'.

Can I create a new pr fix this bug？

**Comment by yesmeck** [comment]
It's obvious, you can not write a function in JSON, how do you plan to fix it?

**Comment by GeoffZhu** [comment]
@yesmeck Back to my previous commit https://github.com/ant-design/babel-plugin-import/commit/3590aca8fda1695cd7243a5e4f256d1780781d5e

``` javascript
if (style === true) {
  addSideEffect(file.path, `${path}/style`);
} else if (typeof style === 'string') {
  addSideEffect(file.path, `${path}/style/${style}`);
}
```

If you don't accept it, I will find another way.

**Comment by yesmeck** [comment]
Though you can not write function in JSON, you can still use function in the config of webpack for babel-loader, so I think it's ok to keep it as it is.

**Comment by GeoffZhu** [comment]
OK, thx

**Pull request merged**